src/components/UserForm/constants.ts

```diff
@@ -0,0 +1,51 @@
+export interface UserFormKeys {
+  username: string;
+  email: string;
+  password: string;
+}
+
+interface InputSections {
+  placeholder: string;
+  type: string;
+  validations: {
+    [key: string]: {
+      value: string | number | boolean | RegExp;
+      message: string;
+    };
+  };
+}
+
+type InputsStructure = Record<keyof UserFormKeys, InputSections>;
+
+export const INPUTS: InputsStructure = {
+  username: {
+    placeholder: 'Username',
+    type: 'text',
+    validations: {
+      required: {
+        value: true,
+        message: "Username is invalid can't be blank"
```

Comment:
Use i18n values, otherwise we'd have errors written in english in the spanish version of the app.